public/app/plugins/panel/table/plugin.json

```diff
@@ -3,6 +3,11 @@
   "name": "Table",
   "id": "table",
 
+  "models": {
+    "version": [1, 0],
+    "changed": "2021/03/30"
```

Comment:
when the models.cue file produces a different models.gen.ts, we should update this date.  Having model version and change date in `panel.json` will let the backend know that this plugin is ready for validation.